src/site/content/en/blog/better-responsiveness-metric/index.md

```diff
@@ -0,0 +1,508 @@
+down and when they lift it up. We need to ensure our metric doesn't count time the user spends
+holding the finger down between these two actions as part of the page's latency! {% endDetails %}
+
+##### Keyboard
+
+A keyboard interaction has two parts to it: when the user presses the key and when they release it.
+There are three associated events with this user interaction: `keydown`, `keyup`, and `keypress`.
+The following diagram illustrates the `keydown` and `keyup` delays and durations for a keyboard
+interaction:
+
+{% Img src="image/kns0INkO77RkiEStzHWYrugyWj32/zjRipPR9ioi551DUa3pj.jpeg", alt="Keyboard interaction
+with disjoint event durations", width="800", height="286" %}
+
+In the diagram above, the durations are disjoint because the frame from `keydown` updates is
+presented before the `keyup` occurs, but this does not need to be the case always. In addition, note
+that a frame can be presented in the middle of a task in the renderer process since the last steps
+required to produce the frame are done outside of the renderer process.
+
+The `keydown` and `keypress` occur when the user presses the key, while the `keyup` occurs when the
+user releases the key. Generally the main content update occurs when the key is pressed: text
+appears on the screen, or the modifier effect is applied. That said, we want to capture the more
+rare cases where `keyup` would also present interesting UI updates, so we want to look at the
+overall time taken.
+
+In order to capture the overall time taken by the keyboard interaction, we can compute the maximum
+of the duration of the `keydown` and the `keyup` events. The `keypress` event is deprecated, and it
+should be fired in the same task as the `keydown` event, so its duration will always overlap with
+the duration of the`keydown`.
+
+{% Details %}
+
+{% DetailsSummary %}
+A note on repeating keypresses
+{% endDetailsSummary %}
+
+There is an edge case here worth mentioning: there may be cases where the user presses a key and
+takes a while to release it. In this case, the sequence of events dispatched can
+[vary](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent#auto-repeat_handling). In
+these cases, we would consider there to be one interaction per `keydown`, which may or may not have
+a corresponding `keyup`. {% endDetails %}
+
+##### Tap
+
+Another important user interaction is when the user taps or clicks on a website. Similar to
+`keypress`, some events are fired as the user presses down, and others as they release, as shown in
+the diagram above, Note the events associated with a tap are a little different on desktop vs
+mobile.
+
+For a tap or click, the release is generally the one which triggers the majority of reactions, but,
+as with keyboard interactions, we want to capture the full interaction. And in this case it’s more
+important to do so because having some UI updates upon tap press is not actually that uncommon.
```

Comment:
I'm not sure this paragraph helps.

For one thing, tap press side-effects are VERY common.  Like every single form element common?

For another, this is backwards from above, which is easy to overlook in the wording, as there end up being a string of logical double negatives (`unlike with the above... the (opposite of the above) is actually import... because its not uncommon`)